Core/Chat/IChatModeChanger.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+
+namespace Core.Chat
+{
+    public interface IChatModeChanger
```

Comment:
If this is only ever going to be used for the commands you could put it in the command collection's file to keep it scoped more tightly and have fewer files

Reply:
Actually maybe not do that, that'd make TwitchChat directly depend on the command